### ISSUE TITLE
aria live bug on info section

### ISSUE THREAD

**Issue by ellvix** [xability/maidr-legacy#265]
aria info section not triggering on same info (such as repeated 'Speed Up' info). 

aria-relevant should be on additions, and we reset the entire div every time. Check if it's not set like that. If it is, check if there's another bug, like aria isn't triggering if the change happens too quickly or something.
